core/designsystem/src/main/kotlin/com/dhc/designsystem/calendar/DhcCalendarController.kt

```diff
@@ -0,0 +1,30 @@
+package com.dhc.designsystem.calendar
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import java.time.LocalDate
+
+@Stable
+class DhcCalendarController(
+    private val initialDate: LocalDate = LocalDate.now(),
+    private val initialPage: Int = Int.MAX_VALUE / 2,
+) {
+    val currentDate = mutableStateOf(initialDate)
+
+    fun getDate(page: Int) = initialDate.plusDays(page.toLong() - initialPage)
```

Comment:
```suggestion
    fun getDateByPage(page: Int) = initialDate.plusDays(page.toLong() - initialPage)
```
Nit : page 를 기반으로 date 를 가져오는것이다보니 함수명을 이렇게 지으면 좀 더 명시적이고 좋을지두?!